types/lint-staged/lint-staged-tests.ts

```diff
@@ -0,0 +1,46 @@
+import { type Config } from 'lint-staged';
```

Comment:
```suggestion
import type { Config } from 'lint-staged';
```
and TS version can drop